### ISSUE TITLE
[sass] output code doesn't change when editing `_global.scss`

### ISSUE THREAD

**Issue by foisonocean** [egoist/rollup-plugin-postcss#83]
`_global.scss` only contains sass variables, and when I try to change the value of those vavibles, output codes won't change.

**Comment by egoist** [comment]
Are you running Rollup in watch mode? currently it's not watched by Rollup.

**Comment by foisonocean** [comment]
yes, in watch mode. Is there any solutions to make `_global.scss` being watched?

**Comment by egoist** [comment]
Seems there's a plugin API planned for this: https://github.com/rollup/rollup/issues/1203

**Comment by egoist** [comment]
Currently I don't have a good solution, you may just use Gulp or [nswatch](https://github.com/egoist/nswatch) watch all your `.scss` files and rerun `rollup` on file change.

**Comment by foisonocean** [comment]
@egoist 
Any updates now? I think the upstream issue was fixed now: https://github.com/rollup/rollup/pull/2259
